depmapomics/expressions.py

```diff
@@ -378,11 +378,11 @@ def postProcess(refworkspace, samplesetname,
     files = subsetGenes(
         files, gene_rename, filenames=trancriptLevelCols, drop="gene_id", index="transcript_id")
 
-  if compute_enrichment:
```

Comment:
keep this version and don't comment that part

Reply:
resolved in new PR